includes/blast_ui.form_advanced_options.inc

```diff
@@ -521,9 +424,6 @@ function blast_ui_blastp_advanced_options_form_submit($form, $form_state) {
   $gapKey = $form_state['values']['gapCost'];
   $gap = _set_protein_gap($matrix, $gapKey);
```

Comment:
set_protein_gap operates on the key (0, 1, 2) but it gets passed the value (ie `11_1`) when submitting blastp job

Reply:
upon futher inpsection, i think that the protein submit wants to be using `get_gap` to explode the string based on the underscore.